sms_no_automatic_delete/tests/test_purge_and_unlink_sms.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Akretion (https://www.akretion.com).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+from odoo.tests.common import TransactionCase
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+class TestSmsPurge(TransactionCase):
+    def setUp(self):
+        super(TestSmsPurge, self).setUp()
+        self.sms_sms = self.env["sms.sms"]
+
+        sms_30_days_ago = datetime.now() - timedelta(days=30)
+        sms_61_days_ago = datetime.now() - timedelta(days=61)
+        sms_91_days_ago = datetime.now() - timedelta(days=91)
+
+        sms_30_days_ago = sms_30_days_ago.strftime(DEFAULT_SERVER_DATE_FORMAT)
+        sms_61_days_ago = sms_61_days_ago.strftime(DEFAULT_SERVER_DATE_FORMAT)
+        sms_91_days_ago = sms_91_days_ago.strftime(DEFAULT_SERVER_DATE_FORMAT)
+
+        self.sms1 = self.sms_sms.create(
+            {
+                "state": "sent",
+                "write_date": sms_30_days_ago,
```

Comment:
Same here pass a datetime directly ;)

```suggestion
                "write_date": datetime.now() - timedelta(days=30),
```

Reply:
indeed, I did it in all files